CMakeLists.txt

```diff
@@ -81,6 +81,7 @@ if(APPLE)
 	set_property(DIRECTORY
 		APPEND
 		PROPERTY COMPILE_DEFINITIONS SC_DARWIN)
+	set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/Install")
```

Comment:
Can you give a reason? Installing to /usr/local is a nuisance to many, I have met somebody who couldn't find SC after installing ;) So what are the risks?